### PR TITLE
Modify miniDST workflow

### PR DESCRIPTION
**Pull request by tmadlener**
BEGINRELEASENOTES
- Rename the `mini-DST-maker.xml` steering file to `MarlinStdRecoMiniDST` and make it a bit easier to configure from the outside
  - Add short description of some available configuration parameters to the miniDST README. 
- Add processors to run angular dEdx correction (see iLCSoft/MarlinReco#91). Make it possible to disable this via a parameter.
- Unpack a default LCFIPlus weights tarball to make it possible to run the miniDST workflow with a minimal default configuration.
- Copy the `IsolatedLeptonTagger` weights to `ILDConfig` from `MarlinReco/Analysis/IsolatedLeptonTagging`
  - Consider removing them from `MarlinReco/Analysis/IsolatedLeptonTagging` as they are "configuration" parameters that should be in `ILDConfig`
  - Easier to use in the miniDST creation example
- Add `run_standard_workflow.sh` for testing of iLCSoft installations
  - Runs 3 evt SIM and RECO steps to produce DST files, which are then also used to make an LCTuple file as well as a mini-DST from it.

ENDRELEASENOTES

### CODE REVIEW

File: StandardConfig/production/MarlinStdRecoMiniDST.xml [hunk 1]
```diff
@@ -817,14 +824,10 @@
...
-  <parameter name="FlavorTag.WeightsDirectory" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/v020201/lcfiweights/4q250_ZZ_v4_p00_ildl5"/>
-  <parameter name="FlavorTag.WeightsPrefix" type="string" value="4q250_v04_p00_ildl5" />
+  <parameter name="FlavorTag.WeightsDirectory" type="string" value="${LCFIPlusWeightsDir}"/>
+  <parameter name="FlavorTag.WeightsPrefix" type="string" value="${LCFIPlusWeightsPrefix}"/>
   <parameter name="FlavorTag.BookName" type="string" value="bdt" />
   <parameter name="FlavorTag.PIDAlgo" type="string" value="lcfiplus" />
-  <!--parameter name="FlavorTag.D0ProbFileName" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/vtxprob/d0probv2_ildl5_6q500.root"/--> <!-- vtxprob/d0probv2_ildl5_6q500.root -->
-  <!--parameter name="FlavorTag.Z0ProbFileName" type="string" value="/afs/desy.de/user/s/skawada/sonas_work/mini-DST/vtxprob/z0probv2_ildl5_6q500.root"/--> <!-- vtxprob/z0probv2_ildl5_6q500.root -->
   <parameter name="FlavorTag.D0ProbFileName" type="string" value="/cvmfs/ilc.desy.de/sw/ILDConfig/v02-02-01/LCFIPlusConfig/vtxprob/d0probv2_ildl5_4q250_ZZ.root"/>
```
Comment (gaede) [inline StandardConfig/production/MarlinStdRecoMiniDST.xml:831]: Could we make these also constants ? 
Could set the path to $PWD and add the corresponding copy and unpack command and how to run it to [https://github.com/iLCSoft/ILDConfig/tree/master/StandardConfig/production#readme](https://github.com/iLCSoft/ILDConfig/tree/master/StandardConfig/production#readme).
This then also serves as documentation and introduction on how to run everything ....

Comment (gaede) [reply StandardConfig/production/MarlinStdRecoMiniDST.xml:831]: ... all necessary files are in `$ILDCONFIG_PATH/LCFIPlusConfig`, i.e. in `../../LCFIPlusConfig` from the current production dir...

File: StandardConfig/production/run_standard_workflow.sh [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    [ -f ${file} ] && rm ${file} || true
+  done
+}
+
+function prepare_lcfiweights() {
+  local lcfiweights=$(pwd)/../../LCFIPlusConfig/lcfiweights
+  local weights=4q250_ZZ_v4_p00_ildl5
+  if [ ! -d ${lcfiweights}/${weights} ]; then
+    echo "Unpacking LCFIPlus weights for test"
+    tar -xzf ${lcfiweights}/${weights}.tar.gz
```
Comment (gaede) [inline StandardConfig/production/run_standard_workflow.sh:38]: ... these are exactly the commands we should add to the README.md

File: StandardConfig/production/MarlinStdRecoMiniDST.xml (hunk 1 above)
Comment (tmadlener) [reply StandardConfig/production/MarlinStdRecoMiniDST.xml]: I have introduced a few more `constants` and tried to give them meaningful defaults, such that minimal example can be run with specifying `ILDConfig_DIR` (assuming unpacked weights)

File: StandardConfig/production/run_standard_workflow.sh (hunk 2 above)
Comment (tmadlener) [reply StandardConfig/production/run_standard_workflow.sh]: Added a few sentences to the README

**Review by gaede — APPROVED** [review]

**Pull request merged**